excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/ChallengeQuestionValidator.java

```diff
@@ -121,16 +124,30 @@ private void validateQuestionId(DefinitionDataItem definitionDataItem) {
 
     private void validateAnswer(DefinitionDataItem definitionDataItem, CaseTypeEntity caseTypeEntity) {
         final String answers = definitionDataItem.getString(ColumnName.CHALLENGE_QUESTION_ANSWER_FIELD);
-        validateNullValue(answers, ERROR_MESSAGE + " value: answer cannot be null.");
-        final String[] answersRules = answers.split(ANSWER_MAIN_SEPARATOR);
-
-        if (answersRules.length > 0 && answersRules.length != 1) {
-            Arrays.asList(answersRules).forEach(currentAnswerExpression ->
-                validateAnswerExpression(definitionDataItem, caseTypeEntity, currentAnswerExpression)
-            );
-        } else {
-            validateAnswerExpression(definitionDataItem, caseTypeEntity, answers);
+        final Boolean ignoreNullFields = definitionDataItem
+                                        .getBoolean(ColumnName.CHALLENGE_QUESTION_IGNORE_NULL_FIELDS);
+        if (!validateEmptyAnswer(ignoreNullFields,answers)) {
+            final String[] answersRules = answers.split(ANSWER_MAIN_SEPARATOR);
+
+            if (answersRules.length > 0 && answersRules.length != 1) {
+                Arrays.asList(answersRules).forEach(currentAnswerExpression ->
+                    validateAnswerExpression(definitionDataItem, caseTypeEntity, currentAnswerExpression)
+                );
+            } else {
+                validateAnswerExpression(definitionDataItem, caseTypeEntity, answers);
+            }
+        }
+    }
+
+    private boolean validateEmptyAnswer(Boolean ignoreNullFields, String answers) {
+        if (answers == null && !ignoreNullFields) {
+            throw new InvalidImportException(ERROR_MESSAGE + " value: answer cannot be null.");
+        }
+        if ((answers == null || answers.isEmpty()) && Boolean.TRUE.equals(ignoreNullFields)) {
+            log.debug("No answer is provided and ignoreNullFields is set to true");
+            return true;
         }
+        return false;
```

Comment:
I think we should be able to revert the changes in this class once other comments are addressed.

Reply:
reverted the changes